staging/src/k8s.io/apiserver/pkg/util/openstack/openstackutils.go

```diff
@@ -323,3 +401,7 @@ func GetNamespaceFromRequest(r *http.Request) string {
 func IsActionRequest(path string) bool {
 	return strings.HasSuffix(path, "action")
 }
+
+func IsBatchCreationRequest(r OpenstackServerRequest) bool {
+	return r.Min_count > 0
```

Comment:
>1?

Reply:
no. 1 will be considered as batch request as well. i.e. a group with one server.

Reply:
The real question is what will be the Min_count value for a non batch request?

Reply:
Please add some comment so that there will be no further similar questions.

Reply:
0, which is the default golang int value will be no-batch cases -- note that in the creation request,  for no-batch case the Openstack request from user ( the json file ) has no min_count, max_count per Openstack spec.  internally we use the same data structure.
the comment are already there in the OpenstackServerRequest{}. i will add some here too.

Reply:
updated.